aws/table_aws_mgn_application.go

```diff
@@ -0,0 +1,202 @@
+package aws
+
+import (
+	"context"
+	"log"
+	"strconv"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/mgn"
+	"github.com/aws/aws-sdk-go-v2/service/mgn/types"
+)
+
+//// TABLE DEFINATION
+
+func tableAwsMGNApplication(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_mgn_application",
+		Description: "AWS MGN Application",
+		List: &plugin.ListConfig{
+			Hydrate: ListAwsMGNApplication,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				// UninitializedAccountException - This error comes up when default template are not set for a particular region.
```

Comment:
Can we provide a doc link for this error?

Reply:
I didn't find any doc for this error, Figured this out during the testing.